AdminWebsite/AdminWebsite/ClientApp/src/app/services/reference-data.service.ts

```diff
@@ -1,14 +1,25 @@
 import { Injectable } from '@angular/core';
-import { BHClient, HearingVenueResponse } from './clients/api-client';
+import { BHClient, HearingVenueResponse, PublicHolidayResponse } from './clients/api-client';
 import { Observable } from 'rxjs';
 
 @Injectable({
     providedIn: 'root'
 })
 export class ReferenceDataService {
+    private publicHolidays: PublicHolidayResponse[];
     constructor(private bhClient: BHClient) {}
 
     getCourts(): Observable<HearingVenueResponse[]> {
         return this.bhClient.getCourts();
     }
+
+    fetchPublicHolidays(): void {
+        this.bhClient.publicHolidays().subscribe({
+            next: pb => (this.publicHolidays = pb)
```

Comment:
do we need next? ive seen it complain before too saying its obsolete but i thought it was because of passing in the object, i think we can do
`this.bhClient.publicHolidays().subscribe(pb => (this.publicHolidays = pb));`
would do the same and not going obsolete? that i know of? i could ofc be wrong about the whole thing

Reply:
i got hit pretty hard in a previous PR not following this else I usually do so like you suggested